create-table.md

```diff
@@ -1,22 +1,111 @@
 ---
 title: CREATE TABLE
 toc: false
+toc_not_nested: true
 ---
 
 The `CREATE TABLE` [statement](sql-statements.html) creates a new table in a database.
 
 <div id="toc"></div>
 
-## Synopsis
+## Syntax
 
 {% include sql/diagrams/create_table.html %}
 
 ## Required Privileges
 
 The user must have the `CREATE` [privilege](privileges.html) on the parent database. 
 
-## Parameters
 
-| Parameter | Description |
-|-----------|-------------|
-|  |  |
\ No newline at end of file
+## Semantics
```

Comment:
Because the sql diagram parameters do not always have specific, helpful names, I've been avoiding this type of list in favor of a "Usage" section. But the parameter names are sufficiently clear in this case, so I understand why you've built in this section. 

For consistency, though, please rename it to "Parameters" and try to do this as a table.